tests/modules/encounters/resources/test_delete_encounter.py

```diff
@@ -38,24 +38,47 @@ def test_delete_method(db, flask_app_client, researcher_1, test_root, staff_user
     assert enc0_id is not None
     assert enc1_id is not None
 
+    # assign indiv to both encounters to test cascade-delete of individual as well
+    indiv_enc_json = {'encounters': [{'id': enc0_id}, {'id': enc1_id}]}
+    response = indiv_utils.create_individual(
+        flask_app_client,
+        staff_user,
+        data_in=indiv_enc_json,
+    )
+    individual_guid = response.json['result']['id']
+    assert individual_guid is not None
+
     ct = test_utils.all_count(db)
     assert ct[0] == orig_ct[0] + 1  # one more sighting
     assert ct[1] == orig_ct[1] + 2  # two more encounters
+    assert ct[4] == orig_ct[4] + 1  # one more individual
```

Comment:
This still feels really clunky, having to know that 0 is sightings, 4 is individual etc. 
This would be better if test_utils.all_count returned an object with sightings and individuals members

Reply:
you mean like keyed off string `sightings` etc?   hmmm yeah, would be clearer.  may throw that into next pr i work on (just cuz i am hoping this sneaks in today... but maybe it is going to need a rebase and rerun tests anyway!  we will see).

Reply:
ok, is now keyed off of class.__name__ so it is pretty much self-documenting in terms of what count you are looking at.

Reply:
Excellent, much clearer